src/server/logger.js

```diff
@@ -65,7 +65,7 @@ class Logger {
         }
 
         // Create color from string hash if available
-        if(chalk.supportsColor.has16m === true){
+        if(chalk.supportsColor.has256 === true){
```

Comment:
would there be a terminal that supports 16m but not 256? in that case this would fail

Reply:
From the looks of https://github.com/chalk/supports-color/blob/8c160ebbf4c8fb3c244295b17972b6a1bf80425f/index.js#L29 , each level of color support is assumed to include the one before it (so 256 color terminals can all do basic ANSI colors, 16m terminals can all do 256 color and basic).